pep-0715.rst

```diff
@@ -21,7 +21,8 @@ deprecating and then disabling new uploads of distribution filenames that have
 the ``.egg`` suffix.
 
 After this PEP, PyPI will only accept new uploads of the ``sdist``
-and ``bdist_wheel`` types.
+and ``bdist_wheel`` types, corresponding to files with ``.zip`` and
+``.whl`` suffixes respectively.
```

Comment:
Err, don't you mean ``.tar.gz`` for sdists, which is by far more popular than `.zip` and produced by all modern backends?

```suggestion
and ``bdist_wheel`` types, corresponding to files with the
``.tar.gz``/``.zip`` and ``.whl`` suffixes respectively.
```

Reply:
Oops, yes. Fixing.

Reply:
1e493d0a